examples/advanced/nlp-ner/utils/bert_ner_model_test.py

```diff
@@ -0,0 +1,195 @@
+# Copyright (c) 2021-2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import os
+import re
+
+import pandas as pd
+import torch
+import torch.nn as nn
+from seqeval.metrics import classification_report
+from torch.utils.data import DataLoader
+from transformers import AutoModelForTokenClassification, AutoTokenizer
+
+os.environ["TOKENIZERS_PARALLELISM"] = "False"
+
+
+class BertModel(nn.Module):
```

Comment:
Seems some code here is duplicated from the custom folder. Can we utilize the same classes to do the testing?

Reply:
This is something I was debating with myself: this script is for testing the performance of the model, so ideally it should be "self-contained" - just give the model and data it should be able to run. Now if we remove this part, we need to first copy the file from jobs/bert_ncbi/app/custom/models/, which looks a strange step before we can perform the testing. @chesterxgchen what's your opinion?

Reply:
With more complex examples that can reuse the code, I would prefer setting the python path to some, as in the [cifar10 example](https://github.com/NVIDIA/NVFlare/blob/dev/examples/advanced/cifar10/cifar10-sim/README.md#1-install-requirements), rather than copying files.

Reply:
yeah makes sense, let me change the readme

Reply:
changed, move custom out of job folder